kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/ConsumerSeekUtilIntegrationTest.java

```diff
@@ -123,7 +123,7 @@ void seekUsingEmptyTokenConsumerStartsAtPositionZero() {
         KafkaTrackingToken testToken = KafkaTrackingToken.newInstance(emptyMap());
 
         Consumer<?, ?> testConsumer = consumerFactory.createConsumer(null);
-        ConsumerSeekUtil.seekToCurrentPositions(testConsumer, () -> testToken, Collections.singletonList(topic));
+        ConsumerSeekUtil.seekToCurrentPositions(testConsumer, () -> testToken, new ListKafkaSubscriber(Collections.singletonList(topic)));
```

Comment:
I think it would be nice to have a test case validating the pattern subscriber to work with the `ConsumerSeekUtils` too.